clang/lib/Sema/SemaTemplateInstantiate.cpp

```diff
@@ -2484,6 +2488,14 @@ TemplateInstantiator::TransformDeclRefExpr(DeclRefExpr *E) {
     if (PD->isParameterPack())
       return TransformFunctionParmPackRefExpr(E, PD);
 
+  if (BindingDecl *BD = dyn_cast<BindingDecl>(D); BD && BD->isParameterPack()) {
+    BD = cast<BindingDecl>(TransformDecl(BD->getLocation(), BD));
```

Comment:
TransformDecl could return null (in which case we should return ExprError())

Reply:
Yep! And this cast will fail here if that is the case.